cl/search/migrations/0089_add_more_courts.py

```diff
@@ -22,6 +22,10 @@ def unload_fixture(apps, schema_editor):
 
 class Migration(migrations.Migration):
 
+    run_before = [
+        ('people_db', '0043_load_tenn_judge_data'),
+    ]
+
```

Comment:
If we keep that migration, I'd set this one as a dependency for that one, rather than tweak an existing migration.